src/services/Room/Room.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  ROOMS_CODES_URL,
+  ROOMS_HOST_URL,
+  ROOMS_URL,
+} from '@/services/apiEndpoint';
+import { axiosAuthInstance } from '@/services/axiosInstance';
+import * as T from '@/types/room';
+
+// 개별 방 정보 조회
+export const getUuidRoom = async (endPoint: string) => {
+  const response: T.RoomResponse = await axiosAuthInstance.get(
```

Comment:
p5)
사소한 부분인데 수현님은 axios 메서드의 응답 타입을 제네릭으로 넘겨주시더라구요!
이렇게 하니까 좀 더 깔끔한 느낌이라 저도 적용했었습니다!
수영님도 axios 함수들에 한 번 적용해보시는 거 어떨까요?

```suggestion
  const response = await axiosAuthInstance.get<T.RoomResponse>(
```